commands.txt

```diff
@@ -1,4 +1,13 @@
 if [ `uname` = 'Darwin' ]; then export SD_CURL_CMD_WRAPPER="eval"; else export SD_CURL_CMD_WRAPPER="$(echo 'sd-step exec --pkg-version 7.54.1 core/curl')"; fi
 cd $SD_ARTIFACTS_DIR
 find . -type f -print > manifest.txt
-find . -type f -exec sh -c 'f=`echo $1 | sed "s/^\.\///"`; encoded=$($SD_CURL_CMD_WRAPPER "curl -s -w \"%{url_effective}\" --data-urlencode \"$f\" -G \"\" | sed -e \"s/^\/?//\""); $SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" -H \"Content-Type: text/plain\" -X PUT $STORE_URL/v1/builds/$SD_BUILD_ID/$ARTIFACTS_DIR_SUFFIX/$encoded -T ./\"$f\""' -- {} \;
+
+if [ "$SD_ZIP_ARTIFACTS" = "true" ]; then
```

Comment:
Lets ensure, we are not adding credentials (eg: aws/.credentials/  or .keys) files in the zip.

Reply:
We are only uploading what is inside `$SD_ARTIFACT_DIR` (https://github.com/screwdriver-cd/artifact-bookend/blob/6909c9f7b91ba6b0a13a977a4c90b55eef593955/commands.txt#L2). There shouldn't be any credentials there unless users intentionally move them there (which I think is user's fault if they do that)